src/app/nebs-database/nebs-database-table/neb-options-filter.pipe.ts

```diff
@@ -18,7 +18,7 @@ export class NebOptionsFilterPipe implements PipeTransform {
     }
     if (kpmValue) {
       filteredOptions = filteredOptions.filter(option => {
-        return option.KPM.includes(kpmValue);
+        return option.KPM.includes(kpmValue) || option.selected;
       })
     } else if (kpiValue) {
```

Comment:
should we do the same to kpiValue filter? including the || option.selected

Reply:
updated thanks